component/src/main/java/io/siddhi/extension/io/http/source/HttpWebSubResponseProcessor.java

```diff
@@ -0,0 +1,298 @@
+                        table.deleteEvents(eventChunk, deleteCompileCondition, 1);
+
+                    }
+                    HttpIoUtil.handleResponse(carbonMessage, HttpIoUtil.createResponseMessageForWebSub(carbonMessage));
+                    sourceEventListener.onEvent(parameterMap, trpProperties);
+                }
+
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Submitted Event " + payload + " Stream");
+                }
+            } else {
+                if (metrics != null) {
+                    metrics.getTotalHttpErrorsMetric().inc();
+                }
+
+                HttpSourceUtil.handleCallback(carbonMessage, 405);
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Empty payload event, hence dropping the event chunk at source " + sourceID);
+                }
+            }
+        } catch (RuntimeException e) {
+            HttpIoUtil.handleFailure(carbonMessage, null, INTERNAL_SERVER_FAIL_CODE, e.getMessage());
+            logger.error("Error occurred while processing the payload ", e);
+        } finally {
+            try {
+                buf.close();
+                carbonMessage.waitAndReleaseAllEntities();
+            } catch (IOException e) {
+                if (metrics != null) {
+                    metrics.getTotalHttpErrorsMetric().inc();
+                }
+
+                logger.error("Error occurred when closing the byte buffer in source " + sourceID, e);
+            }
+        }
+    }
+
+    private Object processAndGet(String value, Attribute attribute) {
+        switch (attribute.getType()) {
+            default:
+            case STRING:
+                return value != null ? value : "N/A";
+            case INT:
+                return Integer.parseInt(value);
+            case BOOL:
+                return Boolean.parseBoolean(value);
+            case LONG:
+                if (attribute.getName().equalsIgnoreCase(HUB_LEASE_SECOND_COLUMN_NAME)) {
+                    return Long.parseLong(value) * 1000;
+                }
+                return Long.parseLong(value);
+            case FLOAT:
+                return Float.parseFloat(value);
+            case DOUBLE:
+                return Double.parseDouble(value);
+        }
+    }
+
+    private void createTableUpdateResources() {
+        Expression condition = generateFilterCondition();
+
+        MetaStateEvent tableUpdateMetaStateEvent = new MetaStateEvent(2);
+
+        MetaStreamEvent tableMetaStreamEvent = new MetaStreamEvent();
+        MetaStreamEvent inputStreamMetaStreamEvent = new MetaStreamEvent();
+
+        TableDefinition inputTableDefinition = TableDefinition.id("");
+        table.getTableDefinition().getAttributeList().forEach((
+                attribute -> inputTableDefinition.attribute(attribute.getName(), attribute.getType())));
+        inputStreamMetaStreamEvent.addInputDefinition(inputTableDefinition);
+
```

Comment:
remove

Reply:
I have added this new line to separate the two tasks for better readability.